src/app/components/ChartbeatAnalytics/canonical/index.test.tsx

```diff
@@ -22,155 +16,13 @@ describe('CanonicalChartbeatAnalytics', () => {
     uid: 123,
   };
 
-  const pageBConfig = {
-    chartbeatConfig: {
-      domain: 'test-domain',
-      type: 'article',
-      sections: 'section1 section2',
-      chartbeatUID: 1111,
-      useCanonical: true,
-      virtualReferrer: '/page-A',
-      title: 'Page B',
-      uid: 123,
-    },
-  };
-
-  const pageCConfig = {
-    chartbeatConfig: {
-      domain: 'test-domain',
-      type: 'article',
-      sections: 'section1 section2',
-      chartbeatUID: 1111,
-      useCanonical: true,
-      virtualReferrer: '/page-B',
-      title: 'Page C',
-      uid: 123,
-    },
-  };
-
   it('should return the helmet wrapper with the script snippet', () => {
     render(
       <CanonicalChartbeatAnalytics
-        chartbeatConfig={pageAConfig}
+        chartbeatConfig={pageConfig}
         chartbeatSource="//chartbeat.js"
       />,
     );
     expect(Helmet.peek().scriptTags).toMatchSnapshot();
   });
-
-  it('should not re-render helment wrapper when config changes to Page B', async () => {
```

Comment:
Just confirming these tests are redundant with support for clientside navigation removed?

Reply:
Yeah I would say so as this was imitating a client side navigation